bangdb/src/main/java/com/yahoo/ycsb/db/BangdbClient.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2016 YCSB contributors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * BangDB client binding for YCSB.
+ *
+ */
+
+package com.yahoo.ycsb.db;
+
+import bangdb.*;
+import com.yahoo.ycsb.ByteIterator;
+import com.yahoo.ycsb.DB;
+import com.yahoo.ycsb.DBException;
+import com.yahoo.ycsb.Status;
+import com.yahoo.ycsb.StringByteIterator;
+
+import java.util.HashMap;
+import java.util.Set;
+import java.util.Vector;
+
+
+
+/**
+ * YCSB binding for <a href="http://bangdb.com/">BangDB</a>.
+ *
+ * See {@code bangdb/README.md} for details.
+ */
+public class BangdbClient extends DB {
+
+  private Connection conn;
+  private Database db;
+  private Table tbl;
+  private TableEnv te;
+  private static boolean flag = true;
```

Comment:
I don't see where this gets used?

YCSB is going to create an instance of this class per client thread, so these static variables, if used, will need locks.